### DISCUSSION TITLE
using hash router but it's not appearing on google analytics

### DISCUSSION THREAD

**Discussion by sombriks** [MatteoGabriele/vue-gtag#428] [Q&A] [answered]
hello all,

i am using vue-gtag 1.16.1 on a vue 2 project  configured like this:

```js
Vue.use(
  VueGtag,
  {
    config: { id: "that-id-thing" },
  },
  router
);
```

my router is using hashmode due to historical reasons 

however, when consulting pages on google analytics i get only the page name, not the hash part:

![Captura de tela de 2022-04-13 23-01-48](https://user-images.githubusercontent.com/556695/163299377-9f8d6c7a-ef81-4e3c-9696-04b733e79025.png)

the blog part loses the path params part, so i can tell which article was visited.

any configuration to allow me to grab the missing part?

that blog page is like this in the router:

```js

  {
    path: "/blog/:post?",
    name: "blog",
    component: () => import(/* webpackChunkName: "blog" */ "../views/Blog.vue"),
  },
``` 

any guidance is welcome.

**Comment by genechuah** [comment]
Hi! Ditto here. Any solution?

**Comment by MatteoGabriele** [comment, marked as answer]
You can use the custom template property and return the values you prefer.
https://matteo-gabriele.gitbook.io/vue-gtag/page-tracker check here for the page tracker template session
This is v3 documentation (similar feature is also available in v2)
